pkg/internal/cluster/providers/docker/provision.go

```diff
@@ -48,6 +48,7 @@ func provision(cluster string, cfg *config.Cluster) error {
 		}
 	}
 	isHA := controlPlanes > 1
+	isIPv6 := cfg.Networking.IPFamily == "ipv6"
```

Comment:
Is there a need to add a `strings.ToUpper` method here?   Allow users to use `IPv6`/`ipv6` etc.

Reply:
No, prefer handling this in the parsing layer.

We do validation and exact strings currently